assets/js/modules/analytics-4/datastore/webdatastreams.js

```diff
@@ -322,6 +322,50 @@ const baseSelectors = {
 	getWebDataStreamsBatch( state, propertyIDs ) {
 		return pick( state.webdatastreams, propertyIDs );
 	},
+
+	/**
+	 * Gets the matched measurement IDs for the given properties.
+	 *
+	 * @since 1.88.0
```

Comment:
Even though we are confident this will be part of the 1.88.0 release, it's still best to use `@since n.e.x.t` and let the version number be added during release preparation.

```suggestion
	 * @since n.e.x.t
```